pwiz_tools/Shared/PanoramaClient/FilePicker.cs

```diff
@@ -0,0 +1,897 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json.Linq;
+using System.Windows.Forms;
+
+
+namespace pwiz.PanoramaClient
+{
+    public partial class FilePicker : Form
+    {
+        private static string _peptideInfoQuery;
```

Comment:
Looks like _peptideInfoQuery is getting build each time a folder node is clicked. So maybe this does not need to be a class variable?